src/sqlfluff/dialects/dialect_tsql.py

```diff
@@ -1577,6 +1662,40 @@ class OverlapsClauseSegment(BaseSegment):
     match_grammar = Nothing()
 
 
+@tsql_dialect.segment(replace=True)
+class DescribeStatementSegment(BaseSegment):
+    """A `Describe` statement.
+
+    Not present in T-SQL.
+    """
+
+    type = "describe_statement"
+    match_grammar = Nothing()
```

Comment:
Rather than have a stub I think you can just remove this from the StatementSegment like this:
![image](https://user-images.githubusercontent.com/80432516/147368729-682334ac-797d-4425-b075-764fbd07d77b.png)

Reply:
@jpy-git thanks for the syntax help!  I moved a couple more stubs to this design now that I know how.